app/src/features/datasets/components/RenderWithHandlebars.tsx

```diff
@@ -0,0 +1,21 @@
+import * as DOMPurify from 'dompurify';
+import 'styles/handlebar.scss';
+import { useHandlebars } from 'utils/handlebarsUtils';
+
+export interface IRenderWithHandleBarProps {
```

Comment:
super nit: looks like we refer to handlebars as one word, so this should be `IRenderWithHandlebarsProps` (lowercase B).

Reply:
I replaved it with:

```export interface IRenderWithHandlebarsProps {
  datasetEML: {
    data: {
      'eml:eml': {
        dataset: any;
        additionalMetadata: any;
      };
    };
  };
  rawTemplate: TemplateSpecification;
}```

Do we know enough about the dataset and addtionalMetadata to go further?

Reply:
We kind of know what to expect. We can look at the EML spec and see which fields are required/optional, etc, and define those in a type. Though there are far too many to define everything.

Additional metadata is optional though, so technically it would need to be treated as an optional field, not sure about dataset.